### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by Mukumbuta**
This PR contains the following:
- Updated README.md file.
- Added MIT.md license file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,226 @@
...
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 [My Debtors] <a name="about-project"></a>
+
+**[My Debtors]** is a Debt Management web application that helps schools to have precise and accurate data of how much their clients owe them.
```
Comment (chimezdev) [inline README.md:29]: **[my_debtors]** is a Debt Management web application that helps schools to have precise and accurate data on how much their clients owe them.

File: README.md [hunk 2]
```diff
@@ -1 +1,226 @@
...
+### Prerequisites
+
+In order to run this project you need:
+[]Frontend
+- Node 
+- npm
+
+[]Backend
+- PIP
+- Python v3.8
```
Comment (chimezdev) [inline README.md:116]: - Python v>=3.8

File: README.md [hunk 3]
```diff
@@ -1 +1,226 @@
...
+
+- GitHub: [@Mukumbuta](https://github.com/Mukumbuta)
+- Twitter: [@Mukumbuta8](https://twitter.com/Mukumbuta8)
+- LinkedIn: [Mukumbuta](https://linkedin.com/in/mukumbuta)
+
+👤 **Chimezie Ochuba**
+
+- GitHub: [@chimezdev](https://github.com/chimezdev)
+- Twitter: [@chimezdev](https://twitter.com/chimezdev)
+- LinkedIn: [chimezdev](https://linkedin.com/in/chimezdev)
```
Comment (chimezdev) [inline README.md:174]: - LinkedIn: [Chimezie Livingstone](https://www.linkedin.com/in/livingstone-chimezie)

**Review by chimezdev — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (Mukumbuta) [reply README.md:174]: Hi @chimezdev 
Thank you for the review

**Comment by chimezdev** [comment]
You did a wonderful one. Thank you

**Pull request merged**